daemon/instructions.rs

```diff
@@ -131,14 +136,23 @@ fn enqueue(message: EnqueueMessage, state: &SharedState) -> Message {
             vec![TaskStatus::Stashed, TaskStatus::Locked],
         );
 
+
         for task_id in &matching {
-            state.change_status(*task_id, TaskStatus::Queued);
+            if message.enqueue_at.is_some() {
+                state.set_enqueue_at(*task_id, message.enqueue_at);
+            } else {
+                state.change_status(*task_id, TaskStatus::Queued);
+            }
         }
 
         (matching, mismatching)
     };
 
-    let message = "Tasks are enqueued";
+    let message = if message.enqueue_at.is_some() {
+        "Tasks enqueueing delayed"
```

Comment:
I think it would be super awesome to get something like:
`Tasks will be enqueued at "Here comes the datetime"`
A good old visual check without calling `pueue status` could be handy.

If you use formatting more often, something like a getter for formatted dates of enqueue, which returns something like `Option<String>` , could be useful. But maybe that's overkill. Do as you wish :)  

Plus `enqueueing` is hell of a word :laughing:

Reply:
Added.